### ISSUE TITLE
Emit last received log event as normal event, after flush_interval has passed?

### ISSUE THREAD

**Issue by mpas** [okkez/fluent-plugin-concat#6]
Currently the last log lines which are in the buffer are emitted as error event after the flush_interval has passed. This results in the fact that i need to place log handling also in the @ERROR label to handle the last message. 

If the flush_interval has passed can the last log event be processed as a normal log line?

An example scenario:
- an application boots up and is just waiting to do something, then the last log line will be emitted as error event. But this is not a real error. It is just doing nothing.

**Comment by okkez** [comment]
You can get timeout errors in your scenario when you use only `multiline_start_regexp`
But you can use both `multiline_start_regexp` and `multiline_end_regexp` if you know what format is your logs.

I will implement timeout events as an arbitrary label, when I understand how to emit such a event.

**Comment by mpas** [comment]
In the current case it is very difficult to determine what the last log line would be, we know for sure what the first line would be. So currently we have setup up a `multiline_start_regexp` that is always consistent. We catch all information between the *start lines*

Example:
```
2014-03-05 10:57:51.112  INFO 45469 --- [           main] org.apache.catalina.core.StandardEngine  : Starting Servlet Engine: Apache Tomcat/7.0.52
2014-03-05 10:57:51.253  INFO 45469 --- [ost-startStop-1] o.a.c.c.C.[Tomcat].[localhost].[/]       : Initializing Spring embedded WebApplicationContext
2014-03-05 10:57:51.253  INFO 45469 --- [ost-startStop-1] o.s.web.context.ContextLoader            : Root WebApplicationContext: initialization completed in 1358 ms
2014-03-05 10:57:51.698  INFO 45469 --- [ost-startStop-1] o.s.b.c.e.ServletRegistrationBean        : Mapping servlet: 'dispatcherServlet' to [/]
```

It would be nice if at certain moment the last log line would be emitted as normal event instead of an error.

**Comment by okkez** [comment]
I've added `timeout_label` option at d8c433102
@mpas could you try it using master HEAD?

For example,

```
<source>
  @type forward
</source>

<filter docker.*>
  @type concat
  key log
  stream_identity_key container_id
  flush_interval 5s
  n_lines 2
  timeout_label @TIMEOUT
</filter>

# handle errors
<label @ERROR>
  <match docker.*>
    @type stdout
  </match>
</label>

# handle timeout events
<label @TIMEOUT>
  <match docker.*>
    @type file
    path /tmp/timeout.log
  </match>
</label>

# in normal case
<match docker.*>
  @type stdout
</match>
```

**Comment by mpas** [comment]
I tested the fix and it seems to work! Thanks for the modification. Please let me know when this gets into a final release.

**Comment by okkez** [comment]
Released v0.5.0
